OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -834,11 +841,11 @@ void testExpressionBasedBushingForce()
         "ball", Vec3(0), Vec3(0), 
         transStiffness, rotStiffness, transDamping, rotDamping);
     
-    spring.setName("linear_bushing");
+    spring.setName("translational_linear_bushing");
     
     osimModel->addForce(&spring);
     
-    osimModel->print("ExpressionBasedBushingForceModel.osim");
+    osimModel->print("TranslationalExpressionBasedBushingForceModel.osim");
```

Comment:
`ExpressionBasedBushingForceModelTranslational.osim` for consistency with `rotational_expression_based_bushing_model_states.sto` on line 1027?